youtube_dl/extractor/daum.py

```diff
@@ -40,11 +40,11 @@ class DaumIE(InfoExtractor):
     }, {
         'url': 'http://m.tvpot.daum.net/v/65139429',
         'info_dict': {
-            'id': '65139429',
+            'id': 'v4e99Kd61HUKxI18xR87xRb',
```

Comment:
It's better to keep the original video ID or ```--download-archive``` will be broken for affected videos. ```smuggle_url``` and ```unsmuggle_url``` can help.